src/xenomods/modules/AppletRedirect.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#include "UpdatableModule.hpp"
+
+namespace xenomods {
```

Comment:
Wrap this namespace in a `#if XENOMODS_CODENAME(bfsw)` as well, otherwise it will crash when attempting to load in other games (as the definition for Initialize isn't declared)